resolwe/flow/executors/__main__.py

```diff
@@ -58,7 +58,7 @@ async def open_listener_connection(data_id, host, port, protocol) -> ZMQCommunic
     """Connect to the listener service."""
     zmq_context = zmq.asyncio.Context.instance()
     zmq_socket = zmq_context.socket(zmq.DEALER)
-    zmq_socket.setsockopt(zmq.IDENTITY, f"e_{data_id}".encode())
+    zmq_socket.setsockopt(zmq.IDENTITY, f"-{data_id}".encode())
```

Comment:
This code is mostly necessary for testing and slurm deploy. 

For listener to differentiate between connected peers they need a different ID.  It is quite fine that init container and communication container use the same ID, since they are not active at the same moment.

On slurm/testing, the executor starts on a possibly different machine, connects to the listener and bootstraps the docker container.  The executor also sends logs to the listener (how much time it took to start the docker container, report exceptions etc) and can therefore be active at the same time as containers. So it must use a different ID. Before I used `e_{data_id}`, but having minus makes code in the listener nicer.